modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientMessagePacker.java

```diff
@@ -26,6 +26,7 @@
 import java.util.BitSet;
 import java.util.UUID;
 import org.apache.ignite.internal.binarytuple.BinaryTupleBuilder;
+import org.msgpack.core.annotations.Nullable;
```

Comment:
Do we use `Nullable` from msgpack?

Reply:
Good catch. Even though it seems to work as expected (IDEA picks up those annotations correctly), I've reverted those changes.